tests/base_d_asa/conftest.py

```diff
@@ -256,11 +257,14 @@ def base_d_asa_client_primary(
 ) -> BaseDAsaClient:
     state = base_d_asa_client_active.state.global_state.get_all()
     base_d_asa_client_active.send.set_secondary_time_events(
-        SetSecondaryTimeEventsArgs(
-            secondary_market_time_events=[state.issuance_date, state.maturity_date]
+        args=SetSecondaryTimeEventsArgs(
+            secondary_market_time_events=[
+                state["issuance_date"],
+                state["maturity_date"],
+            ]
         )
     )
-    utils.time_warp(state.primary_distribution_opening_date)
+    utils.time_warp(state["primary_distribution_opening_date"])
```

Comment:
Same here, I would always use properties approach for consistency in the code base